src/Digdir.Domain.Dialogporten.Application/Features/V1/Common/Content/ContentValueDto.cs

```diff
@@ -5,6 +5,18 @@ namespace Digdir.Domain.Dialogporten.Application.Features.V1.Common.Content;
 
 public class ContentValueDto
 {
+    /// <summary>
+    /// A list of localizations for the content
+    /// </summary>
     public List<LocalizationDto> Value { get; set; } = [];
+
+    /// <summary>
+    /// Media type of the content (plaintext, markdown, html). Can also indicate that the content is embeddable.
```

Comment:
```suggestion
    /// Media type of the content (plaintext, markdown). Can also indicate that the content is embeddable.
```

Planning to only allow HTML for legacy imports, https://github.com/digdir/dialogporten/issues/894